.pylintrc

```diff
@@ -0,0 +1,21 @@
+[MESSAGES CONTROL]
+disable=
+    C0103,
+    R1722,
+    too-many-instance-attributes,
+    missing-module-docstring,
+    invalid-name,
+    expression-not-assigned,
+    unused-argument,
+    attribute-defined-outside-init,
+    missing-function-docstring,
+    too-few-public-methods,
+    fixme,
```

Comment:
Is this just a sane default? I didn't see any instances of `fixme` in the changeset.

Reply:
fixed in https://github.com/macadmins/nudge/pull/49/commits/ab7b5a8062b358c4bf8fa3d5d1d80d360e25e265